pyqtgraph/graphicsItems/ImageItem.py

```diff
@@ -2,13 +2,16 @@
 
 from ..Qt import QtGui, QtCore
 import numpy as np
-import collections
 from .. import functions as fn
 from .. import debug as debug
 from .GraphicsObject import GraphicsObject
 from ..Point import Point
 from .. import getConfigOption
 
+try:
+    from collections.abc import Callable
+except ImportError:
+    from collections import Callable
```

Comment:
I think so. It shouldn't emit a warning for 3.3+. It'll successfully import from `collections.abc` and not try from `collections` as it does now. Python 2.7 will fail first then get it from `collections`.

Reply:
try/except is how numpy/h5py do it (mostly, a few instances of sys.version checks, but that one had other imports that were version dependent as well) 

(Also looked at matplotlib, but mpl master is py36+ only, and thus does not need any checks)

Reply:
@ixjlyons this method would still emit a deprication warning no?

Currently (not this fork)this is the warning I get:

```
pyqtgraph/pgcollections.py:329
  /Users/ognyan/Developer/pyqtgraph/pyqtgraph/pgcollections.py:329: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    class ProtectedList(collections.Sequence):

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

If we are to do the try/except method, would it not make more sense to try to import form `collections` first, and if that fails to import from `collections.abc`?  Doing it in that order should prevent any deprecation warning.